cmd/detectExecuteScan_test.go

```diff
@@ -194,7 +194,7 @@ func TestRunDetect(t *testing.T) {
 		utilsMock.AddFile("detect.sh", []byte(""))
 		err := runDetect(detectExecuteScanOptions{}, utilsMock, &detectExecuteScanInflux{})
 
-		assert.Equal(t, utilsMock.downloadedFiles["https://detect.synopsys.com/detect.sh"], "detect.sh")
+		assert.Equal(t, utilsMock.downloadedFiles["https://detect.synopsys.com/detect7.sh"], "detect.sh")
```

Comment:
Looks like some test cases are missing: you are testing now the default are, that the downloaded file is from detect7.
A test case for an fixed version e.g. 6.8.0 would be good? Question would be what happens if that version is not available anymore, then we would need to adjust the test.

Reply:
Agreed. I will add more test cases. Thanks!